backend/src/report/dto/report-get.dto.ts

```diff
@@ -7,5 +8,6 @@ export class ReportGetDTO {
   public message?: string;
 
   @ValidateNested()
+  @Type(() => ReportFilterDTO)
```

Comment:
fix for startDate before endDate validation not working

referenced https://github.com/typestack/class-validator/issues/83#issuecomment-311663280